docs/product-getting-started-guide/src/main/asciidoc/business-rules-create-proc.adoc

```diff
@@ -0,0 +1,73 @@
+[id='guided-rule-licence-suspension-create-proc']
+= Creating the Driver License Suspension rule
+
+The following chapter instructs you how to create a driver license suspension rule in {PRODUCT} using the Guided Rule wizard.
```

Comment:
See my comment in the Creating the Violation data object section, about intro statements.

Reply:
done